docs/internals/deprecation.txt

```diff
@@ -64,6 +64,9 @@ details on these changes.
 * The undocumented ``django.utils.itercompat.is_iterable()`` function and the
   ``django.utils.itercompat`` module will be removed.
 
+* Support for passing positional arguments to ``Model.save()`` will be
+  removed
```

Comment:
Use a complete sentence.

Reply:
Excuse me for stupid question, but What do you mean by "complete sentence"?

Reply:
Ah, just meant use a period at the end.